drivers/net/ethernet/qualcomm/qca_spi.c

```diff
@@ -810,8 +810,8 @@ qcaspi_netdev_init(struct net_device *dev)
 	qca->clkspeed = qcaspi_clkspeed;
```

Comment:
1. mtu is a mutual parameter, but actually* we need the maximum possible

Reply:
Do you refer to the commit message [e030d2c](https://github.com/chargebyte/linux/pull/3/commits/e030d2c70a5267f7aecd4b3bdf2a81f50bc518fa)

Reply:
yes

Reply:
Fixed